packages/sheet/src/index.ts

```diff
@@ -62,15 +59,18 @@ function createStyleElement(
 }
 
 export class StyleSheet {
-  isSpeedy /*: boolean */
-  ctr /*: number */
-  tags /*: HTMLStyleElement[] */
-  container /*: HTMLElement */
-  key /*: string */
-  nonce /*: string | void */
-  prepend /*: boolean | void */
-  before /*: Element | null */
-  constructor(options /*: Options */) {
+  isSpeedy: boolean
+  ctr: number
+  tags: HTMLStyleElement[]
+  container: HTMLElement
+  key: string
+  nonce: string | undefined
+  prepend: boolean | undefined
+  before: Element | null
+
+  private _alreadyInsertedOrderInsensitiveRule: boolean | undefined
+
+  constructor(options: Options) {
```

Comment:
the previous definition was:
```js
constructor(options?: Options);
```

which was incorrect because `options` are required. 

I'm glad that the whole migration can fix some minor issues like this as isn't done only for the sake of migrating 🎉